css/embedded.css

```
@@ -0,0 +1,25 @@
+/**
+ * @copyright Copyright (c) 2022 Ferdinand Thiessen <rpm@fthiessen.de>
+ *
+ * @license AGPL-3.0-or-later
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+/* Remove background for embedded view */
+body {
+	background-color: var(--color-main-background) !important;
+	background-image: none!important;
+}
```

```suggestion
body {
	background-color: var(--color-main-background) !important;
	background-image: none !important;
}

```

Some small improvements here: missing space and missing new line at the end of the file :)